wrappers/kn/README.md

```diff
@@ -40,6 +42,48 @@ barcodeReader.read(image).joinToString("\n") { barcode: Barcode ->
 
 Here you have to load your image into memory by yourself and pass the decoded data to the constructor of `ImageView`.
 
+### Writing
+
+A trivial use case looks like this:
+
+```kotlin
+import zxingcpp.BarcodeFormat
+import zxingcpp.BarcodeWriter
+import zxingcpp.CreatorOptions
+import zxingcpp.ImageFormat
+import zxingcpp.ImageView
+import zxingcpp.toSVG
+import zxingcpp.toImage
+
+val text: String = "Hello, World!"
+val format = BarcodeFormat.QRCode
+val width: Int = 256
+val height: Int = 256
+
+val opts = CreatorOptions().apply {
```

Comment:
Creating a `CreatorOptions` object without a format does not make any sens. I would like to see a constructor for `CreatorOptions` that forces to supply a format.

Reply:
Rewritten example code, the former one has syntax error: https://github.com/zxing-cpp/zxing-cpp/pull/764/commits/a2cea2de74e1e8dd6b9d29812b3f7851e393f710